.circleci/config.yml

```diff
@@ -0,0 +1,101 @@
+version: 2
+
+.base: &base
+  docker:
+      - image: trojsten/web-base:latest
+      - image: postgres:9.5
+        environment:
+          POSTGRES_USER: trojsten
+          POSTGRES_DB: trojsten
+      # - image: elasticsearch:2.4.6
+  working_directory: ~/web
+
+
+jobs:
+  checkout_code:
+    <<: *base
+    steps:
+      - checkout
+      - save_cache:
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+          paths:
+            - ~/web
+  build_dependencies:
+    <<: *base
+    steps:
+      - restore_cache:
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+      - restore_cache:
+          keys:
+            - v2-dependencies-{{ checksum "requirements.txt" }}
+            # fallback to using the latest cache if no exact match is found
+            - v2-dependencies-
+      - run:
+          name: install dependencies
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip install -r requirements.txt
+            pip install -q coverage
+      - save_cache:
+          paths:
+            - ./venv
+          key: v2-dependencies-{{ checksum "requirements.txt" }}
+  test:
+    <<: *base
+    environment:
+        POSTGRES_DB: trojsten
+        POSTGRES_USER: trojsten
+        POSTGRES_PASSWORD: ""
+        TROJSTENWEB_DATABASE_URL: 127.0.0.1
+        DJANGO_SETTINGS_MODULE: trojsten.settings.test
+        TROJSTENWEB_ELASTICSEARCH_TESTS: 0
+        # TROJSTENWEB_HAYSTACK_CONNECTIONS_URL: http://elasticsearch:9200
+    steps:
+      - restore_cache:
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+      - restore_cache:
+          keys:
+            - v2-dependencies-{{ checksum "requirements.txt" }}
+            # fallback to using the latest cache if no exact match is found
+            - v2-dependencies-
+      - run:
+          name: run tests
+          command: |          
+            . venv/bin/activate
+            sh -c 'cd trojsten; python ../manage.py compilemessages;'
+            coverage run --source='./trojsten' --omit 'trojsten/settings/*,trojsten/special/*' manage.py test
+      - store_artifacts:
+          path: test-reports
```

Comment:
there doesn't seem to be any build artifacts in this directory when I look at the CI output.

Reply:
OK, I'll remove this config for now and add it back when I figure out how to export any artifacts from django test.